BrainPortal/app/models/demand.rb

```diff
@@ -97,15 +72,15 @@ def approved? #:nodoc:
     self.approved_by.present? && self.approved_at.present?
   end
 
-  def is_suspicious?  # types: 1=warning, 2=weird_entries, 3=keyboard_banging
+  def is_suspicious? #:nodoc:
```

Comment:
Yeah, I guess you're right. Now that I think of it, it's not necessary for CBRAIN.

The original NewAccount app was deployed for the bigbrain data access registration of 2013, at the time we were getting > 50 requests per day for months, including a lot of spam. So suspicious highlighting was really useful. But for CBRAIN we don't need that. Feel free to remove all code related to 'suspiciousness'.